### PR TITLE
fix(seed): Format description, add tests and solution for Project Euler 18

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description

Changed the description to more closely match the one on the Project
Euler page. Also added another test and solution.

BREAKING CHANGE: None

Here's a solution for QA:

```
function maximumPathSumI(triangle) {
  let maxSum = triangle.slice();
  
  for (let i = triangle.length - 1; i > 0; i--) {
    let currentRow = maxSum[i];
    let previousRow = maxSum[i - 1];
    const temp = [];
    for (let j = 0; j < i; j++) {
      temp.push(Math.max((currentRow[j] + previousRow[j]), (currentRow[j + 1] + previousRow[j])));
    }
    maxSum[i - 1] = temp;
    maxSum.pop();
  }

  return maxSum[0][0];
}
```

### CODE REVIEW

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json [hunk 1]
```diff
@@ -644,40 +644,46 @@
...
         "  return true;",
         "}",
         "",
-        "euler18();"
+        "const testTriangle = [[3, 0, 0, 0],",
+        "                      [7, 4, 0, 0],",
+        "                      [2, 4, 6, 0],",
+        "                      [8, 5, 9, 3]];",
+        "",
+        "const numTriangle = [[75, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],",
```
Comment (Bouncey) [inline seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json]: Thanks for your patience @scissorsneedfoodtoo, can you move `numTriangle` to the head please, just like before in other challenges.

Many thanks :+1:

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json]: @Bouncey, no problem. I can do that now.

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
Thanks @scissorsneedfoodtoo :+1:

**Comment by scissorsneedfoodtoo** [comment]
@Bouncey, as always, thank you for the QA and merge!

**Pull request merged**